packages/squiggle-lang/benchmark/generators.ts

```diff
@@ -0,0 +1,48 @@
+export let generateFloatRange = (min: number, max: number): number =>
+  Math.random() * (max - min) + min;
+export let generateIntRange = (min: number, max: number): number =>
+  Math.floor(generateFloatRange(min, max));
```

Comment:
What's the reasoning for using ints, instead of floats?

Reply:
I used ints because it would make for smaller cleaner output! If you think that's not worth the change I could change it back to floats (which it was originally)